### PR TITLE
Follow VNC_TYPING_LIMIT everywhere (including send_key)

### PR DESCRIPTION
**Pull request by coolo**
Also lower the default of it, systems are just not tested with that
fast typing speed and lead to unrealistic failures. Fast human typers
can reach 40-50 *words per minute*, which would translate to ~3 keys
per second (a word is averaged to be 5 keys in these type speed contexts).

So trying 50 keys per second is very likely to fail tests that wouldn't
be reproducible in real life. So let's lower this to ten times as fast
as human - and test scenarios that require super fast typing can
increase the VNC_TYPING_LIMIT

Co-Authored-By: Guillaume Gardet <guillaume.gardet@arm.com>

### CODE REVIEW

File: consoles/vnc_base.pm [hunk 1]
```diff
@@ -130,32 +130,35 @@ sub type_string {
             "\e" => 'esc'
         };
         $letter = $charmap->{$letter} || $letter;
-        $self->{vnc}->map_and_send_key($letter);
-        $self->{backend}->run_capture_loop($seconds_per_keypress);
+        # 25% is spent hitting the key, 25% releasing it, 50% seaching the next key
```
Comment (okurz) [inline consoles/vnc_base.pm]: seaching?

Comment (coolo) [reply consoles/vnc_base.pm]: https://context.reverso.net/%C3%BCbersetzung/englisch-deutsch/seach

File: consoles/vnc_base.pm [hunk 2]
```diff
@@ -23,11 +23,15 @@ use feature 'say';
...
 use consoles::VNC;
-use Time::HiRes qw(usleep gettimeofday);
+use List::Util 'max';
+use Time::HiRes qw(usleep time);
 
 use Try::Tiny;
 use testapi 'get_var';
 
+# speed limit: 30 keys per second
+use constant VNC_TYPING_LIMIT_DEFAULT => 30;
```
Comment (okurz) [inline consoles/vnc_base.pm]: Would you mind putting your PR description comment rather in here to explain the value selection? And are you sure we want to loose the speed benefit when "mistyping" now seems to happen only if either modifier keys are involved or it's a clearly special product, e.g. the bootloader line.

File: consoles/vnc_base.pm (hunk 1 above)
Comment (okurz) [reply consoles/vnc_base.pm]: we "search" for the key? Is it a "keyboard from hell" that dynamically changes the layout after each key press? ;)

File: consoles/vnc_base.pm (hunk 2 above)
Comment (coolo) [reply consoles/vnc_base.pm]: I'm not so sure about this anymore. 30 keys a second is still very fast - and we hopefully see less mistypings on e.g. gnome terminals the more realistic our input is.

And I don't believe in code comments to explain numbers - they outdate super fast and the git history is so much more reliable.

**Comment by coolo** [comment]
I don't like splitting the commit

**Comment by okurz** [comment]
fine

File: consoles/vnc_base.pm (hunk 1 above)
Comment (coolo) [reply consoles/vnc_base.pm]: two finger key hunters reach up to 30wbm!

**Pull request merged**